src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -95,6 +95,29 @@ def register_content_settings_argument(self, settings_class, update, arg_group=N
                        'properties listed below is set, then that property will be cleared.',
                        arg_type=get_three_state_flag())
 
+    def register_content_settings_argument_track2(self, settings_class, update, arg_group=None, guess_from_file=None):
```

Comment:
Same here, we can add `process_md5=False` to the original func `register_conetent_settings_argument()` to reduce duplication